abis_mapping/plugins/wkt.py

```diff
@@ -93,7 +93,7 @@ def value_writer(cell: shapely.Geometry) -> str:
             # Serialize to wkt format
             wkt_str = shapely.to_wkt(
                 geometry=cell,
-                rounding_precision=settings.DEFAULT_WKT_ROUNDING_PRECISION,
+                rounding_precision=settings.Settings().DEFAULT_WKT_ROUNDING_PRECISION,
```

Comment:
Every time `Settings()` is instantiated pydantic has to go and read+parse the env variables, could you add `SETTINGS = Settings()` at the bottom of `settings.py` and then reference that everywhere? (similar to how part a does it)

Reply:
I had originally had it that way however pydantic_settings documentation has it being processed in this manner. It makes more sense to reprocess the env variables each time for this library.